test-configs.yaml

```diff
@@ -839,6 +846,9 @@ test_configs:
   - device_type: alpine_db
     test_plans: [boot]
 
+  - device_type: am57xx-beagle-x15
+    test_plans: [boot, boot_nfs, kselftest, simple, sleep]
```

Comment:
I think @khilman suggested to not run `kselftest` on that board in the previous version of this PR.

Reply:
Also please note that only these test plans are listed in `jenkins/lava-boot-v2.sh` (something else that should be put in YAML):
```
boot boot-kvm boot-kvm-uefi boot-nfs boot-nfs-mp simple
```
So any other test plan will not be scheduled in `lab-drue` unless they get added.

I guess `kselftest` should be removed from `test-configs.yaml` and `sleep` should be added to `lava-boot-v2.sh`?

Reply:
Fixed.